mulighetsrommet-arena-adapter/src/main/kotlin/no/nav/mulighetsrommet/arena/adapter/routes/ApiRoutes.kt

```diff
@@ -26,4 +28,19 @@ fun Route.apiRoutes() {
                 )
         call.respond(ExchangeArenaIdForIdResponse(uuid))
     }
+
+    get("/api/status/{id}") {
+        val id = call.parameters["id"]?.toUUID() ?: return@get call.respondText(
+            "Mangler eller ugyldig tiltaksnummer",
```

Comment:
Denne responsen ser ikke helt riktig ut

Reply:
Og det samme gjelder nok for endepunktet over også :sweat_smile: 

(En del endepunkter begynner kanskje å bære litt preg av copy-paste, så kan hende det hadde vært hensiksmessig å ta en runde å rydde opp litt på tvers etter hvert..)